exasol/ds/sandbox/runtime/ansible/roles/entrypoint/files/entrypoint.py

```diff
@@ -75,6 +79,11 @@ def arg_parser():
     return parser
 
 
+def command_with_venv(venv_activate: Path, command: List[str]) -> List[str]:
```

Comment:
This is not needed

Reply:
Ah - left over from an experiment.
Thanks! 
Removed in next push.